Microsoft/WebTemplateStudio
# summary page and modal works

Task:https://microsoftgarage.visualstudio.com/Intern%20GitHub/_sprints/taskboard/WTS%20Team/Intern%20GitHub/Y19-VS/06

This change is to make current summary page (middle panel) as the expected screenshot. (The information originally in summary page is now showing on the right side bar, which is capture by annie's PR).

**How to test:**
1. generate any project type with any page
2. at summary page, clicks on **View Licenses** button, it opens a modal and contains all the corresponding licenses
3. at summary page, clicks on **Give Feedback** button, it should redirect you to GitHub Issue page


**New Summary Page from Designer:** ( the backgroud image in this design is from the internet)
![summary page](https://user-images.githubusercontent.com/12874317/60305807-b5bbc480-98f2-11e9-8f98-7eea2e55d991.png)

**Expected:**
![viewLicenses](https://user-images.githubusercontent.com/12874317/60372189-9f743e00-99b0-11e9-9581-0b994cf0e8b6.PNG)
